src/SimpleGraphs/generators/randgraphs.jl

```diff
@@ -1422,3 +1422,53 @@ function random_orientation_dag(
     end
     return g2
 end
+
+"""
+    bernoulli_graph(Λ)
+
+Given the symmetric matrix ``\\Lambda \\in [0,1]^{n \\times n}``, return a Bernoulli graph with ``n`` vertices. Each edge ``(i,j)`` exists with  probability ``\\operatorname{Bernoulli}(\\Lambda[i,j])``.
```

Comment:
There's a double space: 
```suggestion
Given the symmetric matrix ``\\Lambda \\in [0,1]^{n \\times n}``, return a Bernoulli graph with ``n`` vertices. Each edge ``(i,j)`` exists with probability ``\\operatorname{Bernoulli}(\\Lambda[i,j])``.
```